tardis/io/util.py

```diff
@@ -166,7 +166,91 @@ def check_equality(item1, item2):
         return True
 
 
-def to_hdf(path_or_buf, path, elements, complevel=9, complib='blosc'):
+class HDFReaderWriter():
+
+    @classmethod
+    def to_hdf(cls, path_or_buf, path, elements, complevel=9, complib='blosc'):
```

Comment:
Elements should be attached to the classes as class attributes. Let me know if this is unclear